src/Transfer/Resources/Database/Attributes/EnumAttribute.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Utopia\Transfer\Resources\Database\Attributes;
+
+use Utopia\Transfer\Resources\Database\Attribute;
+use Utopia\Transfer\Resources\Database\Collection;
+
+class EnumAttribute extends Attribute
```

Comment:
```suggestion
class Enum extends Attribute
```


No need of the Attribute suffix. Make sure to change it in the file name.